core/payment/src/error.rs

```diff
@@ -112,7 +112,7 @@ pub mod processor {
     use ya_core_model::payment::public::SendError;
 
     #[derive(thiserror::Error, Debug)]
-    #[error("Account not registered. platform={platform} address={address} mode={mode:?}")]
+    #[error("Account not registered. Hint: Did you run `yagna payment init -r` after restarting? platform={platform} address={address} mode={mode:?}")]
```

Comment:
The flag in the error message should depend on `mode` field. In case of receiving payments it's `-p`.

Reply:
i thought this was not needed since -p is called by default

Reply:
Well, it's called by default but only for the default identity. It is possible to create a new identity and not call init.

Reply:
good one! added the -r or -p dependent on the mode